Formula/magnetix.rb

```diff
@@ -7,6 +7,14 @@ class Magnetix < Formula
 
   depends_on :xcode => :build
 
+  # Port audio code from QTKit to AVFoundation
+  # Required since 10.12 SDK no longer includes QTKit.
+  # Submitted by email to the developer.
+  patch do
```

Comment:
Probably should scope this to `if DevelopmentTools.clang_build_version >= 800`, no?